torch/csrc/distributed/autograd/rpc_messages/rpc_with_autograd.h

```diff
@@ -42,6 +42,8 @@ class TORCH_API RpcWithAutograd final : public rpc::RpcCommandBase {
 
   RpcCommandBase& wrappedRpc();
 
+  std::unique_ptr<RpcCommandBase>&& moveWrappedRpc() &&;
```

Comment:
isn't `std::unique_ptr<RpcCommadBase>` sufficient? Why do we a rvalue reference to the unique pointer itself?

Reply:
right, let me change that